src/map/src/combat/combat.cpp

```diff
@@ -445,9 +445,8 @@ RoseCommon::Entity Combat::get_start_spawn(EntitySystem& entitySystem) {
   auto logger = Core::CLog::GetLogger(Core::log_type::GENERAL).lock();
   logger->trace("Combat::get_start_spawn start");
   for (RoseCommon::Entity entity :
-       entitySystem.get_entities_with_components<Component::BasicInfo, Component::Position, Component::PlayerSpawn>()) {
+       entitySystem.get_entities_with_components<Component::PlayerSpawn>()) {
```

Comment:
You still need `Component::Position` here. This entity gets returned to the calling function get_spawn_point and that expects a `Component::Position`. If you somehow got one that did not have that component, it would cause a crash